interface-definitions/include/firewall/common-rule.xml.i

```diff
@@ -95,6 +95,51 @@
     </constraint>
   </properties>
 </leafNode>
+<node name="ct-status">
+  <properties>
+    <help>Connection status in conntrack</help>
```

Comment:
Maybe just "Connection status". Conntrack is an implementation detail that many users may not really care about, or even know what conntrack is.